docs/pages/distribution/publishing-websites.mdx

```diff
@@ -86,12 +86,23 @@ Learn more about [Netlify](https://www.netlify.com/).
 
 ### Manual deployment with the Netlify CDN
 
-```sh
-npm install netlify-cli -g
-netlify deploy
-```
+1. Install the Netlify CLI by running `npm install netlify-cli -g`
+
+2. Build your Expo web app with `npx expo export:web` (SDK 45 and lower should use `expo build:web` via the legacy global Expo CLI).
+
+3. If your app uses routes you must configure `redirects` so the client has full control over the routing logic:
```

Comment:
```suggestion
2. If your app uses routes, add a **_redirects** file inside the **web-build** directory so the client has complete control over the routing logic. This file should contain the `/* /index.html 200` rule.
```

As per our writing style guide, we prefer to use filenames and directory names as bold text: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text